docs-md/contributing.md

```diff
@@ -32,12 +32,25 @@ We label issues with the ["good first issue" tag](https://github.com/FairwindsOp
 
 The following commands are all required to pass as part of Polaris testing:
 
-```
+```bash
 go list ./... | grep -v vendor | xargs golint -set_exit_status
 go list ./... | grep -v vendor | xargs go vet
 go test ./pkg/... -v -coverprofile cover.out
 ```
 
+### Webhook tests
+```bash
+kind create cluster --wait=90s --image kindest/node:v1.15.11 --name polaris-test
+docker build -t quay.io/fairwinds/polaris:debug .
+docker push quay.io/fairwinds/polaris:debug .
```

Comment:
I'm not sure the `.` on the end is necessary, I get `"docker push" requires exactly 1 argument.` when I run this